common/src/main/java/dev/lavalink/youtube/YoutubeAudioSourceManager.java

```diff
@@ -436,4 +442,29 @@ protected interface Router {
         @Nullable
         AudioItem route(@NotNull Client client) throws CannotBeLoaded, IOException;
     }
+
+    @Nullable
+    public String getRemoteCipherUrl() {
```

Comment:
L446-L468

I don't know how I feel about adding more methods to the main class here. I think these should be part of the cipher manager class. I do see the point to having one function for setting/getting variables, as it simplifies the calling code at the risk of bloating the YoutubeAudioSourceManager class.